spec/index.bs

```diff
@@ -3577,6 +3578,10 @@ offset {{GPUOrigin3D}} in texels, used when copying data from or to a {{GPUTextu
     |textureCopyView|.{{GPUTextureCopyView/texture}}.
   - |textureCopyView|.{{GPUTextureCopyView/origin}}.[=Origin3D/x=] must be a multiple of |blockWidth|.
   - |textureCopyView|.{{GPUTextureCopyView/origin}}.[=Origin3D/y=] must be a multiple of |blockHeight|.
+  - If |textureCopyView|.{{GPUTextureCopyView/texture}}.{{GPUTexture/[[format]]}}
```

Comment:
sadly, this makes "all" rather misleading

Reply:
yes. This means "all" never means more than one in practice, which is misleading, since "all" assumes plural.
This is still fine if we support "all" as in both depth and stencil in some cases.

Reply:
In the case of a D+S texture, `"all"` would really mean "all" (i.e. both the depth and stencil aspects). I think it's somewhat orthogonal whether we actually allow copies of both aspects.

Reply:
Oh yes, not sure why I got confused 😅